src/attributes/views/AttributeCreate/AttributeCreate.tsx

```diff
@@ -104,87 +106,83 @@ const AttributeDetails: React.FC<AttributeDetailsProps> = ({ params }) => {
     setValues(move(values[oldIndex], values, areValuesEqual, newIndex));
 
   return (
-    <AttributeCreateMutation onCompleted={handleCreate}>
-      {(attributeCreate, attributeCreateOpts) => (
-        <>
-          <AttributePage
-            attribute={null}
-            disabled={false}
-            errors={attributeCreateOpts.data?.attributeCreate.errors || []}
-            onBack={() => navigate(attributeListUrl())}
-            onDelete={undefined}
-            onSubmit={input =>
-              attributeCreate({
-                variables: {
-                  input: {
-                    ...input,
-                    storefrontSearchPosition: parseInt(
-                      input.storefrontSearchPosition,
-                      0
-                    ),
-                    values: values.map(value => ({
-                      name: value.name
-                    }))
-                  }
-                }
-              })
+    <>
+      <AttributePage
+        attribute={null}
+        disabled={false}
```

Comment:
Shouldnt we block from when data is loading?

Reply:
Probably